workflow/cron/controller.go

```diff
@@ -39,9 +44,10 @@ type Controller struct {
 	wfInformer         cache.SharedIndexInformer
 	wfLister           util.WorkflowLister
 	wfQueue            workqueue.RateLimitingInterface
-	cronWfInformer     extv1alpha1.CronWorkflowInformer
+	cronWfInformer     informers.GenericInformer
```

Comment:
Why is this change necessary?

Reply:
The generated informers we currently use get "stuck" if a **malformed resource** in created. By "malformed" I mean that while they are correctly formatted as JSON or YAML, they cannot be the resource kind, e.g. because they have an array where an object is expected. This stops the inform working: the informer sits there repeatedly printing an error.

By using a **dynamic informer** (which is effectively the same as as an unstructured informer) we remove this problem and we control the parsing into the required type.